app/services/affiliate-slots-units.json

```diff
@@ -112,5 +112,167 @@
     "link": "https://disneyplus.bn5x.net/DrrRa",
     "country": ["US", "CA", "NL"],
     "isBig": true
+  },
+  {
+    "campaign": "disneyplus",
+    "category": "disney",
+    "image": "https://static.wikia.nocookie.net/a136ba04-c922-40f2-89a6-f455b5ce895a",
+    "heading": "Moana, a heroine kids and adults alike will admire and cheer for",
+    "subheading": "Watch Now On Disney+",
+    "link": "https://disneyplus.bn5x.net/DrrRa",
+    "country": [
+      "AU",
+      "NZ"
+    ],
+    "launchOn": "2019-11-18T22:00:00Z"
+  },
+  {
+    "campaign": "disneyplus",
+    "category": "disney",
+    "image": "https://static.wikia.nocookie.net/a136ba04-c922-40f2-89a6-f455b5ce895a",
+    "heading": "Moana, a heroine kids and adults alike will admire and cheer for",
+    "subheading": "Watch Now On Disney+",
+    "link": "https://disneyplus.bn5x.net/DrrRa",
+    "country": [
+      "AU",
+      "NZ"
+    ],
+    "isBig": true,
+    "launchOn": "2019-11-18T22:00:00Z"
+  },
+  {
+    "campaign": "disneyplus",
+    "category": "mandalorian",
+    "image": "https://static.wikia.nocookie.net/a2ea4f98-6c5b-4cf3-9ee8-e58ec23ba686",
+    "heading": "A thrilling new adventure on the galactic frontier",
+    "subheading": "Watch Now On Disney+",
+    "link": "https://disneyplus.bn5x.net/333RX",
+    "country": [
+      "AU",
+      "NZ"
+    ],
+    "launchOn": "2019-11-18T22:00:00Z"
+  },
+  {
+    "campaign": "disneyplus",
+    "category": "mandalorian",
+    "image": "https://static.wikia.nocookie.net/a2ea4f98-6c5b-4cf3-9ee8-e58ec23ba686",
+    "heading": "A thrilling new adventure on the galactic frontier",
+    "subheading": "Watch Now On Disney+",
+    "link": "https://disneyplus.bn5x.net/333RX",
+    "country": [
+      "AU",
+      "NZ"
+    ],
+    "isBig": true,
+    "launchOn": "2019-11-18T22:00:00Z"
+  },
+  {
+    "campaign": "disneyplus",
+    "category": "marvel",
+    "image": "https://static.wikia.nocookie.net/d8374c50-15b4-4ee6-97be-fa8dc4d5976e",
+    "heading": "All Avengers, All the Time",
+    "subheading": "Watch Now On Disney+",
+    "link": "https://disneyplus.bn5x.net/beeqP",
+    "country": [
+      "AU",
+      "NZ"
+    ],
+    "launchOn": "2019-11-18T22:00:00Z"
+  },
+  {
+    "campaign": "disneyplus",
+    "category": "marvel",
+    "image": "https://static.wikia.nocookie.net/d8374c50-15b4-4ee6-97be-fa8dc4d5976e",
+    "heading": "All Avengers, All the Time",
+    "subheading": "Watch Now On Disney+",
+    "link": "https://disneyplus.bn5x.net/beeqP",
```

Comment:
This also needs to be updated

Reply:
Done https://github.com/Wikia/mobile-wiki/pull/1913/commits/ab61b542efc808b2bd2768eeef6c29dce3977dbe